include/re_tls.h

```diff
@@ -86,6 +92,8 @@ int tls_get_subject(struct tls *tls, struct mbuf *mb);
 void tls_disable_verify_server(struct tls *tls);
 void tls_enable_verify_client(struct tls *tls, bool enable);
 int tls_set_resumption(struct tls *tls, const enum tls_resume_mode mode);
+int tls_set_ocsp_stapling(struct tls *tls, const enum tls_ocsp_stapling mode);
```

Comment:
const is not needed when passing by value ...

Reply:
Thanks. Done.